app/src/main/java/org/fossasia/susi/ai/rest/responses/others/SpeakerAuthResponse.kt

```diff
@@ -0,0 +1,17 @@
+package org.fossasia.susi.ai.rest.responses.others
+
+import com.google.gson.annotations.SerializedName
+
+class SpeakerAuthResponse {
+    @SerializedName("auth")
+    var auth: String? = null
+
+    @SerializedName("authentication")
+    var authentication: String? = null
+
+    @SerializedName("email")
+    var email: String? = null
+
+    @SerializedName("password")
+    var password: String? = null
```

Comment:
Why not val, and SerializedName is not needed when variable name is same as field name